lemur/auth/views.py

```diff
@@ -215,6 +215,8 @@ def update_user(user, profile, roles):
         for ur in user.roles:
             if not ur.third_party:
                 roles.append(ur)
+            else:
+                log_service.audit_log("unassign_role", ur.name, f"Un-assigning the role for {user.name}")
```

Comment:
this might be very verbose, shall only log those which will be truly un-assigned?

like:
```
elif ur not in roles
    log_service.audit_log("unassign_role", ur.name, f"Un-assigning the role for {user.name}")
```

Reply:
That's a good catch! I'll also change this to log single message with a list of removed roles.